### DISCUSSION TITLE
Migrations fail to be applied

### DISCUSSION THREAD

**Discussion by ls-dennis-marchand** [prisma/prisma#10423] [Q&A] [answered]
I've been running into issues where migrations are attempting to be applied to an incorrectly cased table.

```sql
ALTER TABLE `DebitCollectionSummary` DROP COLUMN `currency`;
```

Error received:

```
Error: P3018

A migration failed to apply. New migrations cannot be applied before the error is recovered from. Read more about how to resolve migration issues in a production database: https://pris.ly/d/migrate-resolve

Migration name: 20211124161758_remove_currency_from_debit_summary

Database error code: 1146

Database error:
Table 'redacted.debitcollectionsummary' doesn't exist

Please check the query number 1 from the migration file.
```

I don't understand the database error that I am getting. From what I understand I the migration is trying to run on the table `redacted.debitcollectionsummary` but the migration says it should be run on `redacted.DebitCollectionSummary`. 

This could be helpful for debugging:

```
generator client {
  provider        = "prisma-client-js"
  previewFeatures = ["interactiveTransactions"]
}

datasource db {
  provider = "mysql"
  url      = env("DATABASE_URL")
}
```

**Comment by janpio** [comment, marked as answer]
Can you open a bug report for this please and fill the issue template? We need information like the OS, the Prisma version used and so on to be able to debug this. Thanks.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
